pkg/reconciler/taskrun/resources/taskref.go

```diff
@@ -124,7 +124,21 @@ func GetTaskFunc(ctx context.Context, k8s kubernetes.Interface, tekton clientset
 }
 
 // GetStepActionFunc is a factory function that will use the given Ref as context to return a valid GetStepAction function.
-func GetStepActionFunc(tekton clientset.Interface, namespace string) GetStepAction {
+// also requires a kubeclient, tektonclient, requester in case it needs to find that task in
```

Comment:
this sentence starts out of the blue, was there something else that was required before this?

Reply:
I miss the 'It', sorry. 😢